docs/api.asciidoc

```diff
@@ -207,7 +207,7 @@ For a more in-depth example, see <<log-correlation-manual-unstructured>>.
 [[span-api]]
```

Comment:
In line 193, following the bullets, should they be : or -. In another article I just read they were -

Reply:
Following the bullet, in some of the articles there's a dash used and others it's a colon. So I was suggesting here to make them the same by using a dash instead of a colon.